docs/docs/api/openapi.json

```diff
@@ -1320,7 +1320,7 @@
             "content": {
               "application/json": {
                 "schema": {
-                  "$ref": "#/components/schemas/User"
```

Comment:
You can delete this change.  This was due to a mistake on some automation someone setup.